File: Data Structures & Algorithms/products-of-array-discluding-self/submission-0.go
func productExceptSelf(nums []int) []int {
	someArr := make([]int, len(nums))
	for i := range nums{
		if i == 0{
			someArr[i] = 1
		}else{
			someArr[i] = someArr[i-1]*nums[i-1]
		}
		
	}
	
		for i := range nums{
			product := 1
			for j := i+1; j < len(nums); j++{
				product *= nums[j]
			}
			someArr[i] = someArr[i] * product
		
		}
		
	return someArr
	}

File: Data Structures & Algorithms/concatenation-of-array/submission-0.py
class Solution:
    def getConcatenation(self, nums: List[int]) -> List[int]:
        someArr = [0] * (len(nums)*2)
        for i in range(len(nums)):
            someArr[i] = nums[i]
        for i in range(len(nums)):
            someArr[i+len(nums)] = nums[i]
        return someArr
